### DISCUSSION TITLE
The term 'Get-PnPTeamsApp' is not recognized as the name of a cmdlet, function,

### DISCUSSION THREAD

**Discussion by SanthoshAtMunichRe** [pnp/powershell#1686] [Q&A]
Getting below error when I run the code in Azure Runbook

The term 'Get-PnPTeamsApp' is not recognized as the name of a cmdlet, function,

In my local machine , I have uninstalled all PnP versions and installed 1.9.0 version and getting the below

Method not found: 'Int32 System.Text.Encodings.Web.TextEncoder.FindFirstCharacterToEncodeUtf8(System.ReadOnlySpan`1<Byte>)'.

Please advise on what went wrong. 

Thanks in advance.

**Comment by AndersRask** [comment]
If you run from a runbook, the modules must be added to your automation account (unless you use hybrid groups) Kind regards Anders Rask On 8 Mar 2022 15.19, SanthoshAtMunichRe ***@***.***> wrote:
Getting below error when I run the code in Azure Runbook
The term 'Get-PnPTeamsApp' is not recognized as the name of a cmdlet, function,
In my local machine , I have uninstalled all PnP versions and installed 1.9.0 version and getting the below
Method not found: 'Int32 System.Text.Encodings.Web.TextEncoder.FindFirstCharacterToEncodeUtf8(System.ReadOnlySpan`1)'.
Please advise on what went wrong.
Thanks in advance.

**Comment by SanthoshAtMunichRe** [comment]
the PnP.PowerShell module 1.9.0 version is already added to automation account, still facing the issue.

**Comment by AndersRask** [comment]
Did you upload the modules to the same runtime version that you runbook uses?
Does other pnp cmdlet work fine?
